docker-compose.dev.yml

```diff
@@ -85,3 +85,49 @@ services:
     restart: "no"
   sparql-authorization-wrapper:
     restart: "no"
+  besluiten-consumer-0:
+    environment:
+     # #### START: 'normal operation mode' in optimal performance for direct virtuoso (needs full-reindex though)
+      #SLEEP_BETWEEN_BATCHES: 1 # also maybe increast this during intial sync
+      #BATCH_SIZE: 1000
+      #BYPASS_MU_AUTH_FOR_EXPENSIVE_QUERIES: 'true'
+      #PARALLEL_CALLS: "4" # we might be jinxing virtuoso; but let's try
+      # #### END: 'normal operation mode' in optimal performance for direct virtuoso
+
+      # #### START: 'normal operation mode' in optimal performance for mu-auth
+      BATCH_SIZE: "1"
+      SLEEP_BETWEEN_BATCHES: "1"
+      PARALLEL_CALLS: "8"
```

Comment:
This is the only consumer service that is using parallel calls with value 8 instead of 16. Do we change it to 16 so evcerything is inline? This so when we look at it later that we do not question ourself why is this one different than the other 3 services.

Reply:
We will:
- move these L100 - L102 to the compose file
- change the parallel calls to 4 as we define `ThreadsPerQuery` equal to `4` in the virtuoso.ini config